php/libraries/Utility.class.inc

```diff
@@ -350,6 +350,41 @@ class Utility
         return $result;
     }
 
+    /**
```

Comment:
@driusan I had to add this because we don't have a function that reurns visits associated to the project-subproject tuple (even though that is what defines visits) 

This also DOES NOT return proper visit objects because the visitclass does not properly define the getVisitFromName but instead defines getVisitsFromName which is incorrect and I am fixing in a futur PR.